Formula/buildozer.rb

```diff
@@ -14,11 +14,96 @@ class Buildozer < Formula
     sha256 cellar: :any_skip_relocation, high_sierra: "a257921eb9df552c485cb2656ce7f535a708846308de4aa1ee47942492a61aee"
   end
 
-  depends_on "bazelisk" => :build
+  depends_on "go" => :build
 
   def install
-    system "bazelisk", "build", "--config=release", "buildozer:buildozer"
-    bin.install "bazel-bin/buildozer/darwin_amd64_stripped/buildozer"
+    ENV["GOPATH"] = buildpath
+    ENV["GO111MODULE"] = "auto"
+    commit = Utils.git_short_head
+    ldflags = ["-X main.buildVersion=#{version}",
+               "-X main.buildScmRevision=#{commit}"]
+    dir = buildpath/"buildozer"
+    (dir/"go.mod").write <<~EOS
+      module github.com/bazelbuild/buildozer
+      go 1.15
+      require (
+        github.com/bazelbuild/buildtools v0.0.0-20210208125002-752270cf2f52
+        github.com/golang/protobuf v1.4.3 // indirect
+        google.golang.org/protobuf v1.25.0 // indirect
+      )
+    EOS
+
+    (dir/"go.sum").write <<~EOS
```

Comment:
We do not want to maintain these in the formula, please submit them upstream.

Reply:
Then please remove buildozer, buildifier and charm-tools from being tested when a new go version is released.  
Go 1.15.8 is being held hostage by packages that fail because of unrelated reasons to go.